### ISSUE TITLE
Possible memory leak

### ISSUE THREAD

**Issue by aheze** [aheze/Popovers#20]
I presented a bunch of popovers by rapidly clicking each button. Here's the memory report:

![Screen Shot 2022-01-29 at 6 08 54 PM](https://user-images.githubusercontent.com/49819455/151683996-b9693b4d-30ad-466b-8917-236d53233ecd.png)

The memory usage just keeps going up! It seems like a new `PopoverContainerView` is created every time you present a popover.

To test, I added this code inside `PopoverContainerView`:
```swift
if #available(iOS 15.0, *) {
    let _ = Self._printChanges()
}
```

![Screen Shot 2022-01-29 at 6 09 11 PM](https://user-images.githubusercontent.com/49819455/151683992-683fe0ea-913f-43d9-a907-f96e89583884.png)

The result was `PopoverContainerView: _popoverModel changed.` being printed hundred of times.

**Comment by aheze** [comment]
Yeah this doesn't seem right

![Screen Shot 2022-01-29 at 6 27 04 PM](https://user-images.githubusercontent.com/49819455/151684381-686ce826-1b2b-4272-98e2-4d5e41109937.png)

**Comment by aheze** [comment]
So I have malware installed on my simulator then? Interesting

![Screen Shot 2022-01-29 at 6 33 28 PM](https://user-images.githubusercontent.com/49819455/151684509-6d94f779-15b0-4b86-98e7-36150fcfe19a.png)

Jkjk I've almost never used the memory graph before

**Comment by aheze** [comment]
This makes the Container view deinit. But `Context` still never gets deinited...
![Screen Shot 2022-01-29 at 6 44 30 PM](https://user-images.githubusercontent.com/49819455/151684701-a956b192-7007-4beb-b17d-1ef4add10465.png)

**Comment by aheze** [comment]
So it seems like there are 4 places that hold a reference to `Context`:

![Screen Shot 2022-01-29 at 7 04 39 PM](https://user-images.githubusercontent.com/49819455/151685415-c3cacde0-e9ab-4046-9192-877a625706f2.png)

Backtrace 4:
![Screen Shot 2022-01-29 at 7 20 18 PM](https://user-images.githubusercontent.com/49819455/151685423-0242558c-16f6-4f04-90b8-6ef9306dabb8.png)

Backtrace 5:
![Screen Shot 2022-01-29 at 7 20 30 PM](https://user-images.githubusercontent.com/49819455/151685434-e4a75dac-014c-4913-91df-b06f56a61611.png)

Backtrace 7:
![Screen Shot 2022-01-29 at 7 20 56 PM](https://user-images.githubusercontent.com/49819455/151685440-d806e5a3-6f06-42ac-8f78-a0d410827303.png)

Backtrace 8:
![Screen Shot 2022-01-29 at 7 21 12 PM](https://user-images.githubusercontent.com/49819455/151685445-e3dce663-33a6-46ea-9c04-c513c44649d3.png)


Here's the memory graph:
![Screen Shot 2022-01-29 at 7 19 36 PM](https://user-images.githubusercontent.com/49819455/151685406-06199f6f-8511-4183-9f74-b4001e79c18c.png)

**Comment by aheze** [comment]
Fixed in 217350047c848f03044e7e5ab7cc6a79d71f20b4! Kind of dumb — I forgot to remove the reference to the popover inside the SwiftUI modifier.

![Screen Shot 2022-02-03 at 6 56 18 PM](https://user-images.githubusercontent.com/49819455/152464807-73013f89-6c34-494e-b12a-63434f3f5b40.png)

**Comment by aheze** [comment]
Will publish the fix in v1.2.1.

**Comment by aheze** [comment]
Fixed in [1.3.0](https://github.com/aheze/Popovers/releases/tag/1.3.0).
